include/taskolib/SequenceManager.h

```diff
@@ -124,32 +141,52 @@ class SequenceManager
      * \exception Error is thrown if the specified folder name is not valid or if it does
      *            not contain a valid sequence.
      */
-    Sequence load_sequence(std::filesystem::path sequence_folder) const;
+    Sequence load_sequence(UniqueId uid) const;
+
+    /**
+     * \copydoc load_sequence(UniqueId)
+     *
+     * This overload allows to pass in a list of sequences to avoid having to examine the
+     * base path again:
+     *
+     * \param sequences  a list of sequences as obtained from list_sequences()
+     */
+    Sequence load_sequence(UniqueId uid, const std::vector<SequenceOnDisk>& sequences)
+        const;
```

Comment:
Maybe it is better to have the return type in a single line instead of the `const` so lonely:
```c++
Sequence
load_sequence(UniqueId uid, const std::vector<SequenceOnDisk>& sequences) const;
```

Reply:
Fixed, pending force-push.